src/components/Presentation/AnimationEditor/EffectEditor/index.jsx

```diff
@@ -1,11 +1,52 @@
+import { useContext } from "react";
 import { styled } from "styled-components";
+import { useParams } from "react-router-dom";
+import { useQueryClient, useMutation } from "@tanstack/react-query";
+import { ObjectContext } from "../../../../Contexts/ObjectContext";
+import axiosInstance from "../../../../services/axios";
```

Comment:
```suggestion
import { useParams } from "react-router-dom";
import { useQueryClient, useMutation } from "@tanstack/react-query";
import { styled } from "styled-components";
import { ObjectContext } from "../../../../Contexts/ObjectContext";
import axiosInstance from "../../../../services/axios";
```
import 순서변경 부탁 드립니다 🙇‍♂️

Reply:
감사합니다~ 알려주신 내용 으로 수정 했습니다! 확인 부탁드립니다~